### PR TITLE
Conform _AppsDrawer.pcss to the naming policy - AppDrawer variants

### PR DESCRIPTION
**Pull request by luixxiul**
For https://github.com/vector-im/element-web/issues/25268

This is one of the PRs which intend to conform `_AppsDrawer.pcss` to our naming policy by clarifying that the `mx_AppsDrawer--maximised`, `mx_AppsDrawer--resizing`, etc. are the variants of `mx_AppsDrawer`.

type: task

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

## Checklist

-   [ ] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Review by gsouquet — COMMENTED** [review]
Minor comment regarding the naming of a CSS property, but that should otherwise be good to go.

File: res/css/views/rooms/_AppsDrawer.pcss [hunk 1]
```diff
@@ -96,7 +92,15 @@ limitations under the License.
         min-width: var(--minWidth);
     }
 
-    &.mx_AppsDrawer_2apps .mx_AppTile {
+    &.mx_AppsDrawer--maximise {
```
[the remark this answers is not in the document]
Comment (luixxiul) [reply res/css/views/rooms/_AppsDrawer.pcss:95]: I'll update the corresponding code too.

[the remark this answers is not in the document]
Comment (luixxiul) [reply res/css/views/rooms/_AppsDrawer.pcss]: Addressed with https://github.com/matrix-org/matrix-react-sdk/pull/10856/commits/43d7f4fba084f158a58c7315fc4cd1528d7c1c31.

**Review by gsouquet — APPROVED** [review]

**Comment by luixxiul** [comment]
@gsouquet would you kindly add the PR to the queue?

**Pull request merged**